reupdate the changes for the RTTTL component (#3320)
OttoWinter/esphomedocs: components/rtttl.rst

    # Example configuration entry
    output:
      - platform: ...
        id: rtttl_out
        ...

    rtttl:
      output: rtttl_out
      id: my_rtttl

Overview Using the I2S speaker
------------------------------

> The tone generator can instead be used with a :doc:`Speaker </components/speaker>` to output the audio.

Reviewer: ```suggestion
The tone generator can instead be used with a :doc:`Speaker </components/speaker/index>` to output the audio.
```

    # Example configuration entry
    output:

>       - platform: ledc

Reviewer: ```suggestion
      - platform: ...
```

        id: rtttl_out
        ...

    rtttl:
      output: rtttl_out
      id: my_rtttl

Overview Using the I2S speaker
------------------------------

> Using the tone generator with the :ref:`I2S Speaker <speaker>` the following code sample is needed.
> :doc:`ESP32 speaker </components/speaker>` is supported.

Reviewer: ```suggestion
The tone generator can instead be used with a :doc:`Speaker </components/speaker>` to output the audio.
```

.. code-block:: yaml

    # Example configuration entry

>     speaker:
>       - platform: i2s_audio
>         id: my_speaker
>         ...
> 

Reviewer: ```suggestion
    speaker:
      - platform: ...
        id: my_speaker
        ...

```

See Also
--------
- :doc:`/components/output/esp8266_pwm`
- :doc:`/components/output/ledc`

> - :doc:`/components/speaker`

Reviewer: ```suggestion
- :doc:`/components/speaker/index`
```
